Assets/MRTK/Core/Inspectors/Setup/MixedRealityEditorSettings.cs

```diff
@@ -115,5 +125,16 @@ private static void LogConfigurationWarnings()
                 }
             }
         }
+
+        /// <summary>
+        /// Checks critical project settings and suggests changes to optimize build performance via logged warnings
+        /// </summary>
+        private static void LogBuildConfigurationWarnings()
+        {
+            if (PlayerSettings.stripUnusedMeshComponents)
+            {
+                Debug.LogWarning("<b>Optimize Mesh Data</b> is enabled. This setting can drastically increase build times. It is recommended to disable this setting during development and re-enable during \"Master\" build creation. See <i>Player Settings</i> > <i>Other Settings</i> > <i>Optimize Mesh Data</i>");
```

Comment:
Looks good. I would again recommend updating MRTK documentation and possibly add link to this error message as well. "For more info see {blah}"

Reply:
Good idea! Added a blub in the docs.